chamberse/encounters/mpg_ingenuity.lua

```diff
@@ -11,107 +14,142 @@
-    eq.zone_emote(MT.Yellow, "You have " .. minutes_remaining .. " minutes remaining to complete your task.");
-  end
+	if e.message:findi("hail") and not event_active then
+		e.self:Say("This is the Mastery of Ingenuity trial. You must demonstrate your ability to think on your feet, to defeat an unbeatable opponent. Are you ready to [begin]?");
+	elseif e.message:findi("begin") and not event_active then
+		local dz = eq.get_expedition()
+		if dz.valid then
+			dz:SetLocked(true, ExpeditionLockMessage.Begin, 14) -- live uses "Event Messages" type 365 (not in emu clients)
+		end
+
+		eq.spawn_condition(this_zone,instance_id,1,1);
+		e.self:SetSpecialAbility(SpecialAbility.immune_aggro, 0);
+		e.self:SetSpecialAbility(SpecialAbility.immune_aggro_on, 0);
+		e.self:Say("Very well!  Let the battle commence!");
+		eq.set_timer("minutes", 1 * 60 * 1000);
+		eq.set_timer("reflective", 45 * 1000);
+		eq.set_timer("ds_clear", 5 * 1000);
+		eq.set_timer("pet_clear", 3 * 60 * 1000);
+		eq.zone_emote(MT.Yellow, "You have " .. minutes_remaining .. " minutes remaining to complete your task.");
+		event_active = true;
+	end
 end
 
 
 function Ingenuity_Timer(e)
-  if (e.timer == "minutes") then 
-    minutes_remaining = minutes_remaining - 1;
-
-    -- If the minutes reach 0; then the Master of Ingenuity has not died; 
-    -- otherwise we would have stopped the timer.
-    if (minutes_remaining == 0) then
-      eq.stop_all_timers();
-      eq.spawn_condition('chamberse', instance_id, 1, 0);
-
-      eq.zone_emote(MT.Red, "You have been found unworthy.");
-      eq.depop();
-
-      -- no lockout added on failure, the dz shuts down in 5 minutes
-      local dz = eq.get_expedition()
-      if dz.valid then
-        dz:SetSecondsRemaining(eq.seconds("5m"))
-      end
-    else 
-      eq.zone_emote(MT.Yellow, "You have " .. minutes_remaining .. " minutes remaining to complete your task.");
-    end
-  elseif (e.timer == "reflective") then
-    -- Spell: Reflective Sheen - 5822 
-    e.self:Emote("begins to cast a spell.  <Reflective Sheen>");
-    e.self:CastSpell(5822, e.self:GetID()); -- Spell: Reflective Sheen
-  end
+	if e.timer == "minutes" then 
+		minutes_remaining = minutes_remaining - 1;
+
+		-- If the minutes reach 0; then the Master of Ingenuity has not died; 
+		-- otherwise we would have stopped the timer.
+		if minutes_remaining == 0 then
+			eq.stop_all_timers();
+			eq.spawn_condition(this_zone,instance_id,1,0);
+
+			eq.zone_emote(MT.Red, "You have been found unworthy.");
+			eq.depop();
+
+			local dz = eq.get_expedition()
+			if dz.valid then
+				dz:AddReplayLockout(eq.seconds(lockout_loss))
+				dz:SetSecondsRemaining(eq.seconds("5m"))
+			end
+		else 
+			eq.zone_emote(MT.Yellow, "You have " .. minutes_remaining .. " minutes remaining to complete your task.");
+		end
+	elseif e.timer == "reflective" then
+		-- Spell: Reflective Sheen - 5822 
+		e.self:Emote("begins to cast a spell.  <Reflective Sheen>");
```

Comment:
I think this is because on live it force listed it in-era as many people still didn't have spell message leader aas and without it you would have no way of knowing it was casting? I will chat with Huffin on this, we may nuke it and rely on people setting up audio triggers for the on-cast spell message.